modin/core/execution/ray/common/utils.py

```diff
@@ -74,6 +76,13 @@ def initialize_ray(
     """
     extra_init_kw = {"runtime_env": {"env_vars": {"__MODIN_AUTOIMPORT_PANDAS__": "1"}}}
     if not ray.is_initialized() or override_is_cluster:
+        # TODO(https://github.com/ray-project/ray/issues/28216): remove this
+        # workaround once Ray gives a better way to suppress task errors.
+        # Ideally we would not set global environment variables.
+        # If user has explicitly set _RAY_IGNORE_UNHANDLED_ERRORS_VAR, don't
+        # don't override its value.
+        if _RAY_IGNORE_UNHANDLED_ERRORS_VAR not in os.environ:
```

Comment:
What if ray is already initialized by the user? Do we want supress errors in remote tasks in that case?

Reply:
Initially I thought we wouldn't want to change the user's ray setup if they've already set up ray, but now I think because we should always change the variable because it seems from user code I see that people often initialize ray first.

Reply:
So I changed it; please take a look

Reply:
Maybe we should set the env var after ray is initialized? Imagine the case if ray re-set the env var during ray.init(...) even if we call it.

Reply:
Done.